### DISCUSSION TITLE
Tailwind ignores some styles for a project with Nextjs

### DISCUSSION THREAD

**Discussion by tengkuzulfadli** [tailwindlabs/tailwindcss#5972] [Help]
Hi,

I'm having an issue now;

--> When I set a style for a sm screen, it changes to all screen sizes.
--> I tried to override with a plain css or even using classes, still not working
--> I reinstalled using the latest version of Nextjs and Tailwindcss, still not working

How can I solve this issue?

**Comment by simonswiss** [comment]
Hey!

Tailwind CSS uses a "mobile-first" approach, meaning that the base, unprefixed classes are applying to all breakpoints by default.

When you use a responsive variant like `text-red-500 md:text-blue-500`, you will apply `color: blue` to every breakpoint _from `sm` and up_. So `md`, `lg`, `xl`, ... will also be blue, and that's the expected behaviour.

This is because Tailwind's default breakpoints are using `min-width` media queries.

If you want the `md` breakpoints to use a different color for that element, you need to change that color for the breakpoint you're targeting:

```html
text-red-500 md:text-blue-500 lg:text-green-500
```

Hope it helps!

If you need more specific assistance, please share some code in a Tailwind Play so when can understand what exactly you need help with 👍
